### PR TITLE
Add tests for empty output directory

### PR DESCRIPTION
**Pull request by vermakhushboo**
## What does this PR do?

Need to release OPR version for this to pass

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work. Screenshots may also be helpful.)

## Related PRs and Issues

- (Related PR or issue)

## Checklist

- [ ] Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?
- [ ] If the PR includes a change to an API's metadata (desc, label, params, etc.), does it also include updated API specs and example docs?

## Summary by CodeRabbit

- **Tests**
	- Enhanced test coverage to ensure deployments fail gracefully with clear error notifications when the build output directory is empty or missing.
	- Added tests for scenarios where the output directory exists but is empty, and where the output directory does not exist at all.
	- Updated existing tests to validate more specific error messages in deployment logs.

### CODE REVIEW

File: tests/e2e/Services/Sites/SitesCustomServerTest.php [hunk 1]
```diff
@@ -2535,7 +2535,66 @@ public function testEmptySiteSource(): void
...
+        }, 100000, 500);
+
+        $this->cleanupSite($siteId);
+    }
+
+    public function testOutputDirectoryMissing(): void
+    {
+        $siteId = $this->setupSite([
+            'siteId' => ID::unique(),
+            'name' => 'Empty output directory',
```
Comment (Meldiron) [inline tests/e2e/Services/Sites/SitesCustomServerTest.php:2578]:
```suggestion
            'name' => 'Missing output directory',
```

**Review by Meldiron — APPROVED** [review]
